.github/workflows/electron-win.yml

```diff
@@ -0,0 +1,43 @@
+name: Electron Rebuild CI - Windows (x32 & x64)
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        electron:
+          - 5.0.0 #70
+          - 6.0.0 #73
+          - 7.0.0 #75
+          - 8.0.0 #76
+        arch:
+          - ia32
+          - x64
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        os: ${{ matrix.os }}
+    - name: Node Ver (For debugging)
+      run: node -v
+    - name: Install packages
+      run: npm install --unsafe-perm
+    - name: Run on ${{ matrix.os }} - Electron v${{ matrix.electron }}
+      run: node ".\node_modules\node-gyp\bin\node-gyp.js" rebuild --target=${{ matrix.electron }} --arch=${{ matrix.arch }} --dist-url=https://electronjs.org/headers
+    - uses: actions/upload-artifact@v1
+      with:
+        name: win32_${{ matrix.arch }}_${{ matrix.electron }}.node
```

Comment:
How can I replace `${{ matrix.electron }}` with the node version commented above (line 13-16)